.gitignore

```diff
@@ -1,3 +1,5 @@
+tests/openwisp2/saml/*
```

Comment:
what does this has to do with this PR?

Reply:
These changes are related to my local environment which accidentally got added to this PR. I have removed these changes from the PR.